plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -65,6 +67,11 @@ public IcebergTableProperties(IcebergConfig icebergConfig)
                         "File system location URI for the table",
                         null,
                         false))
+                .add(booleanProperty(
+                        HIVE_ENABLED,
+                        "Enable hive support",
```

Comment:
The preference is for setting this as an environment property -- your Hive install should have the Iceberg Jar or not. But you may want to set it at the table level for some tables because you can add the Iceberg runtime Jar on the client side. I would probably not include this in Trino unless you need to later.

Reply:
What exactly you mean by this ? "I would probably not include this in Trino unless you need to later". Keep the table property as it is ? Or change to as config variable ? I tried to do the same thing as it is done in spark iceberg jars, when creating table compatibile with hive appropriate table property is assigned.
Or you mean that chnage at all shouldnt be included in trino ?

Reply:
I wouldn't expose this in Trino unless it is necessary. People can set this in the HiveConf used by the catalog, so hopefully you can just ignore these settings and make that the way to control it. Hopefully Hive will have a better story around this soon as well, so we don't need to use fake InputFormat and Serde classes to avoid it breaking. Normally, I wouldn't expect needing to do so much in other engines to avoid Hive's user experience from breaking.

Reply:
Do you have any idea how to achieve that on table level ? I mean at the moment only way is to create table using spark or hive to be able to use on those 3 engines. I mean Hive, Trino, Spark.

Reply:
Iceberg has the same logic in the code, I mean by using table property.

Reply:
> I wouldn't expose this in Trino unless it is necessary. People can set this in the HiveConf used by the catalog,

@rdblue you mean hive-site.xml files (trino's `hive.config.resources` config)? 
or you mean some HMS configuration change? 

thanks for confirming we don't need this to be a table property.

Reply:
Yeah, Iceberg will also check for that property to be configured. I'd leave it in Hive config rather than using session config or table config.